service/RTC/SignalingLayer.ts

```diff
@@ -96,23 +98,27 @@ export default class SignalingLayer extends Listenable {
      * @param {MediaType} mediaType the type of the media for which presence
      * @param {SourceName} sourceName - The name of the source for which the info is to be obtained.
      * info will be obtained.
-     * @return {PeerMediaInfo|null} presenceInfo an object with media presence
+     * @return {IPeerMediaInfo|null} presenceInfo an object with media presence
      * info or <tt>null</tt> either if there is no presence available for given
      * JID or if the media type given is invalid.
      *
      * @deprecated This method is to be replaced with getPeerSourceInfo.
      */
-    getPeerMediaInfo(owner, mediaType, sourceName) { // eslint-disable-line no-unused-vars
+    getPeerMediaInfo(
+            owner: string, mediaType: MediaType, sourceName: SourceName
+    ): IPeerMediaInfo | null { // eslint-disable-line no-unused-vars
```

Comment:
You can now remove this and prefix the parameters with an underscore and it should be silent.